css/style.css

```diff
@@ -1,7 +1,285 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+}
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+body {
+	height: 100vh; /*значение высоты опр-ся по viewport*/
+	margin: 0 auto; /*выравнивание по центру*/
+	background: url(../img/iswanto-arif-448003-unsplash.png) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
+	color: #E8EEE4; /*цвет шрифта*/
+	font: normal 22px/26px MuseoSansCyrl; /*стиль шрифта*/
+	overflow: scroll;
+}
+
+/*header styles start*/
+.container {
+	max-width: 1650px; 
+	max-height: 180px;
+	margin: 0 auto; /*выравнивание по центру*/
+}
+
+.header__row { 
+	display: flex; 
+	align-items: flex-end; /*вертикальное выравнивание по нижнему краю контейнера*/
+	justify-content: space-between; /*выравнивание по ширине - по краям конейнера*/
+	width: 100%;
+	margin: 0 5%;
+	margin-bottom: 9.5%;
+}
+
+.header__logo {
+	padding-top: 6%;
+	width: 146px;
+}
+
+.header__logo img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.header__menu {
+	display: flex;
+	justify-content: space-around;
+	margin-right: 6%;
+}
+
+.menu__link {
+	flex-wrap: nowrap;
+	color: #E8EEE4;
+	margin-right: 33.75px; /*отступ справа у пуктов меню*/
+	text-decoration: none; /*убрали нижнее подчеркивание*/
+}
+
+.menu__link:hover, .menu__link:focus, .menu__link:active { /*стили для наведения на ссылки меню*/
+	opacity: .50;
+}
+
+.header__burger { /*бургер-меню*/
+	display: none;
+	width: 25px;
+	height: 17px;
+	position: relative;
+}
+
+.burger__line {
+	display: block;
+	width: 100%;
+	height: 3px;
+	background: #FCF5ED;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#FCF5ED" to be "#fcf5ed" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455739)